tokio-net/tests/signal_drop_multi_loop.rs

```diff
@@ -1,8 +1,8 @@
 #![cfg(unix)]
+#![cfg(feature = "signal")]
 #![warn(rust_2018_idioms)]
 
-use libc;
-
+#[path = "support/signal.rs"]
```

Comment:
Maybe we can call this module `signal_support` to avoid having to put the `path` attribute everywhere